### PR TITLE
#259 - Use external recommender URL from trait

### PR DESCRIPTION
**Pull request by Rentier**

### CODE REVIEW

**Comment by reckart** [comment]
@Rentier if there is a PR, there is an issue. When merging the PR, use "squash" which allows you to change the commit message and to enter the number of the PR.

File: inception-imls-external/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/ExternalClassifier.java [hunk 1]
```diff
@@ -108,9 +108,11 @@ public void reconfigure()
         }
         
         //Contruct Http Request
-        // TODO: Use traits url
-        String remoteUrl = "http://localhost:12889/tag";
-        // String remoteUrl = traits.getRemoteUrl();
+        String remoteUrl = traits.getRemoteUrl();
+        if (remoteUrl == null || remoteUrl.isEmpty()) {
```
Comment (reckart) [inline inception-imls-external/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/ExternalClassifier.java]: Use Apache Commons `StringUtils.isBlank()` ;)

File: inception-imls-external/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/ExternalClassifier.java [hunk 2]
```diff
@@ -108,9 +108,11 @@ public void reconfigure()
...
+            remoteUrl = "http://localhost:12889/tag";
```
Comment (reckart) [inline inception-imls-external/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/ExternalClassifier.java]: Can we not encode magic default values please?

**Comment by Rentier** [comment]
@reckart I thought about it and think that we should not set a default value in the classifier.

**Pull request merged**